apps/condo/domains/ticket/hooks/useTableColumns.tsx

```diff
@@ -184,6 +204,18 @@ export function useTableColumns <T> (filterMetas: Array<FiltersMeta<T>>) {
         (assignee) => getTableCellRenderer(search)(get(assignee, ['name'])),
         [search])
 
+    const { user } = useAuth()
+
+    const ticketIds = useMemo(() => tickets.map(ticket => ticket.id), [tickets])
```

Comment:
Simpler form using `lodash`:
```suggestion
    const ticketIds = useMemo(() => map(tickets, 'id')), [tickets])
```